src/Deck.java

```diff
@@ -0,0 +1,542 @@
+// Name: Cristian Vazquez, Justin Thomas, Marcos Barrera, Michael Janes
+// Class: CST 338 - Module 3
+// Assignment: Decks of Cards
+// Date: 22 March 2020
+
+import java.util.Random;
+
+// Phase 3: The Deck Class
+public class Deck
+{
+   public static void main(String[] args)
+   {
+      // Test of Class Deck (from phase 3)
+      
+      System.out.println("----------- 2 Decks (Unshuffled) ------------------");
+      Deck doubleDeck = new Deck(2);
+      
+      int i = 0;
+      while (i < doubleDeck.cards.length)
+      {
+         System.out.println( doubleDeck.cards[i].toString() );
+         i += 1;
```

Comment:
another short hand version this would be
i++;

This evaluates i, then adds 1 to it.
Not to be confused with ++i;